development-updates.md

```diff
@@ -494,3 +495,7 @@ Development updates from [`WillQ`](https://github.com/zsluedem/)
 ### WillQ: Update 2
 
 [Update #2](https://hackmd.io/@WillQ/H19FwIyHj)
+
+### WillQ: Update 3
+
+[Update #3](https://hackmd.io/@WillQ/BJc2-CKSj)
```

Comment:
403 error, doc is not available. Please make sure it's published

Reply:
Oh. Sorry. Fixed!